controllers/emailService.js

```diff
@@ -0,0 +1,39 @@
+import sgMail from '@sendgrid/mail';
```

Comment:
As I said in the earlier comment, create a new directory named `services` and put this email service in it.

Controllers typically have `req` and `res` and this does not have. Also, you can take out the error part since we established that a wrong email never gets the mail and the app does not need to error out.